cli/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "starknet-rs-cli"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["with_mimalloc"]
+with_mimalloc = ["mimalloc"]
+
+[dependencies]
+cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "4afacfd574de1121d61dea28f8c73e1516c7e07d"}
+cairo-vm = { git = "https://github.com/lambdaclass/cairo-rs", features = ["cairo-1-hints"] }
+getset = "0.1.2"
+lazy_static = "1.4.0"
+num-bigint = { version = "0.4", features = ["serde"] }
+num-integer = "0.1.45"
+num-traits = "0.2.15"
+serde = { version = "1.0.152", features = ["derive"] }
+serde_json = { version = "1.0", features = ["arbitrary_precision"] }
+sha3 = "0.10.1"
+starknet_api = { git = "https://github.com/lambdaclass/starknet-api", rev="b29d881", features = [
+    "testing",
+] }
+starknet-crypto = "0.4.3"
+thiserror = "1.0.32"
+clap = { version = "4.1.8", features = ["derive"] }
+actix-web = "4.3.1"
+awc = "3.1.1"
+mimalloc = { version = "0.1.29", default-features = false, optional = true }
+hex = "0.4.3"
+cargo-llvm-cov = "0.5.14"
+starknet-rs = {path = "../"}
```

Comment:
I think you should not use a relative path to specify the dependency but the github repository